pandas/core/internals/managers.py

```diff
@@ -366,6 +367,37 @@ def reduce(
             new_mgr = type(self).from_blocks(res_blocks, [self.items, index])
         return new_mgr, indexer
 
+    def operate_scalar(self, other, op) -> BlockManager:
```

Comment:
other: Scalar?

Reply:
Can we use `Scalar` in this case? In principle it can be any python scalar, not only the set of pre-defined scalar types for `Scalar` in our typing module (but I don't know how this is used elsewhere in pandas, so it might be we ignore this possibility elsewhere as well)

Reply:
i dont think Scalar corresonds to lib.is_scalar (there is an issue about this IIRC).  i think it also can include not-really-scalars for object-dtype.  kind of "anything than can be a fill_value"